src/SourceLocator/SourceStubber/PhpStormStubsSourceStubber.php

```diff
@@ -33,6 +33,8 @@
 use function str_replace;
 use function strtolower;
 
+use const PHP_VERSION_ID;
```

Comment:
If I understand correctly, this uses "the PHP version BR is running under", rather than "the PHP version of the code being analysed" as the default? What is the mechanism for overriding this? e.g. if we are using PHP 8.0 to analyse a codebase that runs on PHP 7.4?

Reply:
You can always specify the version by yourself: https://github.com/Roave/BetterReflection/pull/874/commits/419c1a72cbfaa72e0a02afd7c565c0c79470e0e5#diff-462651b2de3f83cc239bebc471d6800cebf5893e7258569982934b031c8c422eR169